kolibri/content/api.py

```diff
@@ -27,10 +27,6 @@ class ChannelMetadataFilter(filters.FilterSet):
     available = filters.django_filters.MethodFilter()
 
     def filter_available(self, queryset, value):
-        if value == "true":
```

Comment:
This filter is still being used in the learn tab, please leave intact, and it will still be making a GET param `"true"` value, not a Python boolean.

Reply:
Jonathan is only changing the content import/export part but not the learn app. I can put back the if statement inside the filter function. I was just thinking that the filter is not used anywhere else except the learn.html, so it seems to be more clean to revert it back to where it was before (not having the if statement here). But maybe we need if statement to filter "True" and "False" values for future uses?